lib/ukatomic/Config.uk

```diff
@@ -0,0 +1,5 @@
+menuconfig LIBUKATOMIC
+	bool "ukatomic: Atomic operations"
+	default y
+	help
+	  Support for atomic operations.
```

Comment:
The linux kernel uses two spaces to indent `help`, and personally I prefer that too, but the Unikraft convention is to use a tab here too, so let's stick to that 🙂